src/app/pages/OnDemandRadioPage/index.jsx

```diff
@@ -4,12 +4,21 @@ import styled from 'styled-components';
 import path from 'ramda/src/path';
 import MetadataContainer from '../../containers/Metadata';
 import Grid, { GelPageGrid } from '#app/components/Grid';
-import OnDemandRadio from '#containers/RadioPageBlocks/Blocks/OnDemandRadio';
 import { ServiceContext } from '../../contexts/ServiceContext';
 
+import HeadingBlock from '#containers/RadioPageBlocks/Blocks/Heading';
+import ParagraphBlock from '#containers/RadioPageBlocks/Blocks/Paragraph';
+
+const SKIP_LINK_ANCHOR_ID = 'content';
+
 const OnDemandRadioPage = ({ pageData }) => {
+  const blocks = path(['content', 'blocks'], pageData);
+  const episodeTitle = path(['0', 'title'], blocks);
```

Comment:
This might not be a thing for right now - and might be worth speaking to @aeroplanejane & @gavinspence about it - I would like it if we could "translate" the auto-generated episode title into a useful localised date

If the summary is in the format "dd/mm/yyyy GMT", then could we try to convert it to a date, and then render the timestamp component instead. e.g. **04/02/2020 GMT** would be converted to the equilvalent of **4th February 2020** in the language of that particular service. Happy to discuss.